src/infer_request.cc

```diff
@@ -175,6 +176,20 @@ InferRequest::Trace()
   return trace_;
 }
 
+uint32_t
+InferRequest::ReleaseFlags()
+{
+  request_release_flags_ = infer_request_shm_ptr_->request_release_flags;
```

Comment:
Question for my learning: why do we use a temporary variable here and in set release flags?

Is that needed because of shared memory?